memphis/consumer.py

```diff
@@ -229,11 +248,22 @@ async def main(host, username, password, station):
     async def __ping_consumer(self, callback):
         while True:
             try:
+                
                 await asyncio.sleep(self.ping_consumer_interval_ms / 1000)
+                station_inner = get_internal_name(self.station_name)
                 consumer_group = get_internal_name(self.consumer_group)
-                await self.connection.broker_connection.consumer_info(
-                    self.station_name, consumer_group, timeout=30
-                )
+                if not self.inner_station_name in self.connection.partition_consumers_updates_data:
```

Comment:
Please change to `if self.inner_station_name not in self.connection.partition_consumers_updates_data`